cobigen-templates/templates-devon4j/src/main/templates/crud_openapi_net/templates/Devon4Net.WebAPI.Implementation/Configure/DevonConfiguration.cs

```diff
@@ -1,5 +1,6 @@
 using System;
 using Devon4Net.WebAPI.Implementation.Business.TodoManagement.Service;
+using Devon4Net.WebAPI.Implementation.Business.EmployeeManagement.Service;
```

Comment:
Is `Employee` here correct? or is it because you forgot to add a FreeMarker variable?